### ISSUE TITLE
Codium 1.89.1 refuses to start on Windows 11 x64 machine

### ISSUE THREAD

**Issue by KagaJiankui** [Alex313031/codium#10]
**Describe the bug**
Codium 1.89.1 release for Windows x64 downloaded from the release page of this repo refuses to start after freshly installation.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
1. Install the Codium 1.89.1 System Installer of the newest release.
2. Launch Codium executable.
3. The window stuck at a blank dark grey status.

**Expected behavior**
Codium Start normally.

**Screenshots**
![image](https://github.com/Alex313031/codium/assets/58453886/237ea81b-df2b-48b8-8fa1-e06d1c591ffb)

**Desktop (please complete the following information):**
 - OS: Windows 11 10.0.22631
 - Architecture x64
 - Version 1.89.1
 - App Manager no
 - Sandboxed no

**Comment by gz83** [comment]
Does 1.89.0 work properly?

**Comment by Alex313031** [comment]
@KagaJiankui I mean, the point of this fork is to allow running on old Windows OSes. Why are you wanting to run Codium on Win 11 when you could use VSCodium?

**Comment by KagaJiankui** [comment]
Maybe expect for some better performance as appealed by compile-time optimizations.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by spacedrone404** [comment]
Weird request, win11 is a complete abomination.

**Comment by Alex313031** [comment]
@spacedrone404 Agreed. I will see if I can reproduce, but because the entire point of this fork is to make a compiler optimized VSCodium, with a non-ugly logo, that works on old Windows and Linux (Primarily targeting Windows 7/8, and Ubuntu 18.04/Debian 10), I won't be investing too much time into this if I can't figure it out easily.

You could just run VSCodium on Win 11.

**Comment by Alex313031** [comment]
@KagaJiankui @spacedrone404 Can you try the latest 1.93.1 release on win11?

**Comment by KagaJiankui** [comment]
I'm out of reach to my machine as I'm outside on vacation for a couple days. Will try the newest version later ASAP.
________________________________

**Comment by spacedrone404** [comment]
> @KagaJiankui @spacedrone404 Can you try the latest 1.93.1 release on win11?

sorry, but i am not using win10/11

**Comment by Alex313031** [comment]
@spacedrone404 Hehe what are you using?

And Windows 11 24H2 and 23H2 are not playing well with virtualbox, so I can't test it without the test being potentially inaccurate since it would be on an older 2022 version of Win11

**Comment by spacedrone404** [comment]
> @spacedrone404 Hehe what are you using?
> 
> And Windows 11 24H2 and 23H2 are not playing well with virtualbox, so I can't test it without the test being potentially inaccurate since it would be on an older 2022 version of Win11

I am using Win 7 x64 ESU + some arch based Linux distros
